packages/teams-js/test/internal/appIdValidation.spec.ts

```diff
@@ -78,3 +80,43 @@ describe('validateStringAsAppId', () => {
     expect(() => validateStringAsAppId('hello\u0080world')).toThrowError(/printable/i);
   });
 });
+
+describe('validateStringAsAppId', () => {
+  test('should not throw for "valid" app ids', () => {
+    expect(() => validateStringAsAppId('8e6523aa-97f9-49ad-8614-75cae22f6597')).not.toThrow();
```

Comment:
Are these `validateStringAsAppId` tests just adding some missing coverage? They don't look like they're testing the new validation, right?

Reply:
duplicated tests were unintentional. I have deleted those now!